toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -234,42 +234,6 @@ func (b *LiveOSIsoBuilder) populateWriteableRootfsDir(sourceDir, writeableRootfs
 	return nil
 }
 
-// prepareRootfsForDracut
-//
-//	ensures two things:
-//	- initrd image build time configuration is in place.
-//	- rootfs (squashfs) image contents are compatible with our LiveOS initrd
-//	  boot flow.
-//	note that the same rootfs is used for both:
-//	(1) creating the initrd image and
-//	(2) creating the squashfs image.
-//
-// inputs:
-//   - writeableRootfsDir:
-//     root directory of existing rootfs content to modify.
-//
-// outputs:
-// - all changes will be applied to the specified rootfs directory in the input.
-func (b *LiveOSIsoBuilder) prepareRootfsForDracut(writeableRootfsDir string) error {
```

Comment:
Why was this function removed?

Reply:
The steps in this function (creating a dracut config and deleting fstab) are really part of creating the initrd image. So, I've moved the two steps into that createInitrdImage() function at the top.